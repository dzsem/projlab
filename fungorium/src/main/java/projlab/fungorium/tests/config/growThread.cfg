# Grow Thread
# ------------------------------------------
# Init state
# ------------------------------------------
# 0
ADD TECTON
# 1
ADD TECTON
REGISTER NEIGHBOUR 0 1
# 2
ADD MUSHROOMBODY 0 0
# 3
ADD MUSHROOMTHREAD 0 0