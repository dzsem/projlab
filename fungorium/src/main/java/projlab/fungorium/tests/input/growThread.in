# Grow Thread
# ------------------------------------------
# Init state
# ------------------------------------------
# 0
ADD TECTON
# 1
ADD TECTON
REGISTER NEIGHBOUR 0 1
# 2
ADD MUSHROOMBODY 0 0
# 3
ADD MUSHROOMTHREAD 0 0
# Input
# ------------------------------------------
# 4, t0 -> t1
MUSHROOMTHREAD 3 GROW 1
# Print states
# ------------------------------------------
LIST TECTON
LIST MUSHROOMTHREAD