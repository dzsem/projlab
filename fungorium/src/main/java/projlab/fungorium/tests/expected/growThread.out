TECTON 0 1 2 1 0
TECTON 1 1 -1 1 0
MUSHROOMTHREAD 3 0 0 1 UNCUT SPROUT
MUSHROOMTHREAD 4 0 1 1 UNCUT SPROUT